app/code/Magento/Customer/Model/FileProcessor.php

```diff
@@ -215,7 +215,9 @@ public function saveTemporaryFile($fileId)
         );
 
         $result = $uploader->save($path);
-        unset($result['path']);
+        if (is_array($result)) {
```

Comment:
Solution require discussion

Reply:
Hello @xmav 
https://wiki.php.net/rfc/autovivification_false - this is reason of this solution for PHP 8.1.
The system fails if $result = false without this checking.
Also, pay attention, `$uploader->save($path)` can return boolean value

Reply:
I changed the solution, please check.